modules/i18n/src/translations/en-US/portals/users.ts

```diff
@@ -288,6 +288,20 @@ export const users: usersNS = {
                 message: "Role retrieval successful"
             }
         },
+        impersonateUser: {
+            error: {
+                description: "Couldn't impersonate user.",
+                message: "Error impersonating user"
+            },
+            genericError: {
+                description: "Couldn't impersonate user.",
+                message: "Something went wrong"
+            },
+            success: {
+                description: "Successfully impersonated the user.",
```

Comment:
this alert is shown when an impersonation session is started, right? if that's the case, let's update the alert description to reflect that.

Eg:

```suggestion
                description: "User impersonation started successfully.",
```

Reply:
Resolved with e566b44176dd034f329c9cfb9c4a4d39093c7f56